sBTC SIP

```diff
@@ -0,0 +1,166 @@
+- **Stacks 2.5 Participation:** Active participation running a signer on Stacks 2.5 testnet or mainnet for at least two previous cycles.
+- **Technical Performance and Uptime:** Consistency in operational status and network stability. Signers should demonstrate a target uptime of at least 99%.
+- **Communication & Availability:** Running an sBTC Signer is a pivotal role in the Stacks ecosystem, which requires a high degree of availability and communication with Stacks core developers. Signers must be able to respond to updates within 24 hours.
+- **Ecosystem Alignment:** Commitment to the growth and principles of the Stacks and Bitcoin ecosystems with contributions to advance the network.
+
+By implementing a whitelisted signer set for the sBTC Bootstrap release, this helps to reduce time to market to launch sBTC sooner, remove complexity from the initial protocol design, and ensure the sBTC protocol is secure at launch.
+
+### Activation
+sBTC Bootstrap is designed to activate on Stacks Nakamoto as defined in SIP-021. Therefore, this SIP is only meaningful when SIP-021 activates. The sBTC Working Group plans to observe at least 2-4 weeks of network behavior on Stacks Nakamoto to ensure a stable release. After this period, sBTC Bootstrap can be activated on the Stacks network without requiring a separate hard fork.
+
+**Process of Activation**
+Users can vote to approve this SIP with either their locked/stacked STX or with unlocked/liquid STX, or both. The criteria for the stacker and non-stacker voting is as follows.
+
+**For Stackers:**
+In order for this SIP to activate, the following criteria must be met by the set of Stacked STX:
+
+- At least 80 million Stacked STX must vote at all to activate this SIP.
+- Of the Stacked STX that vote, at least 80% of them must vote "yes."
+
+The voting addresses will be;
+
+Bitcoin Yes Address: 3Jq9UT81fnT2t24XjNVY7wijpsSmNSivbK
+Bitcoin No Address: 3QGZ1fDa97yZCXpAnXQd6JHF4CBC6bk1r4
+Stacks Yes Address: SP36GHEPEZPGD53G2F29P5NEY884DXQR7TX90QE3T
+Stacks No Address: SP3YAKFMGWSSATYNCKXKJHE2Z5JJ6DH88E4T8XJPK
+
+which encode the hashes of the following phrases into bitcoin / stacks addresses:
+
+Yes to A Decentralized Two-Way Bitcoin Peg
+No to A Decentralized Two-Way Bitcoin Peg
+
+Stackers (pool and solo) vote by sending a dust stacks to the corresponding stacks address from the account where their stacks are locked.
+
+Solo stackers only, can also vote by sending a bitcoin dust transaction (6000 sats) to the corresponding bitcoin address.
+
+**For Non-Stackers:**
+Users with liquid STX can vote on proposals using the Ecosystem DAO. Liquid STX is the users balance, less any STX they have locked in PoX stacking protocol, at the block height at which the voting started (preventing the same STX from being transferred between accounts and used to effectively double vote). This is referred to generally as "snapshot" voting.
```

Comment:
Link to Ecosystem DAO page for voting?

Reply:
@andrerserrano i think this comment is still relevant, i believe it's just a link to stx.eco?

Reply:
Link to voting page was added.